### PR TITLE
Init terminologies map

### PR DESCRIPTION
**Pull request by mohamed7genius**
* Add rollup-json
* Add full list of terminologies words with explanation
* Make sure that any attached special char before/after the terminology is added again after making the terminology and the explanation clickable.
* Add space before and after `/` when two terminologies are used ( because we are slicing the words based on spaces)

### CODE REVIEW

File: src/components/Content/terminologies.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "makrooh": "disliked",
+  "mustahabb": "recommended",
+  "nifaas": "postnatal bleeding",
+  "qadaa": "makeup",
+  "salaah": "prayer",
+  "shari'iyyah": "Islamic law",
+  "tawaaf": "circumambulation",
+  "tuhr": "purity",
+  "umrah ": "minor pilgrimage",
+  "waajib": "necessary",
```
Comment (Ajwah) [inline src/components/Content/terminologies.json:17]:
```suggestion
  "wajib": "necessary",
```

**Review by Ajwah — COMMENTED** [review]
Where are the markdown changes? You need to delete the bracket long-format in all the markdown files

File: src/components/Content/terminologies.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "ma’dhur": "excused person",
+  "makrooh": "disliked",
+  "mustahabb": "recommended",
+  "nifaas": "postnatal bleeding",
+  "qadaa": "makeup",
+  "salaah": "prayer",
+  "shari’iyyah": "Islamic law",
+  "sunnah": "the teachings, practices, and examples set forth by the Prophet SAW",
+  "takbeer-e-tahreema": "the opening phrase of the prayer",
+  "tawaaf": "circumambulation",
```
Comment (Ajwah) [inline src/components/Content/terminologies.json:18]: circumambulation around the Ka'ba

Comment (mohamed7genius) [reply src/components/Content/terminologies.json:18]: I used the spelling in our terminologies `kabah`

**Review by Ajwah — APPROVED** [review]

**Pull request merged**